osmdroid-android/src/main/java/org/osmdroid/tileprovider/modules/MapTileFilesystemProvider.java

```diff
@@ -32,7 +32,7 @@ public class MapTileFilesystemProvider extends MapTileFileStorageProviderBase {
     // Fields
     // ===========================================================
 
-    private final TileWriter mWriter = new TileWriter();
+    private TileWriter mWriter = new TileWriter();
```

Comment:
Please leave it `final`, for code clarity.
If needed, put a default value in constructors.

Reply:
Good point, thanks.